### ISSUE TITLE
Wrong scaling of bounding boxes if I do not use default --img 640

### ISSUE THREAD

**Issue by sandreel** [ultralytics/yolov5#1593]
## ❔Question

Thanks you for providing Yolov5, it looks very promising! 

I have an issue with wrongly scaled bounding boxes, I will explain it here using coco images.

If I follow the "Train Custom Data" tutorial (https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data) for training on coco images (with the default --img 640), everything works as expected. However if I change the training resolution of the images (the example below was created with --img 1024) the boxes in the **validation** batches are not in the right places anymore. 

![test_batch2_label_coco_1024](https://user-images.githubusercontent.com/32390578/101019519-64931400-356d-11eb-99c1-b2fc6c84f54c.jpg)

Also, if I split the coco128 images into a train and a validation set, the model does not appear to be learning anything (actually the mAP is decreasing during the training, which makes sense as I started from a pre-trained model).

![map](https://user-images.githubusercontent.com/32390578/101020577-049d6d00-356f-11eb-87ac-64c1196b5d76.PNG)

The model was trained with:

**python train.py --img 1024 --batch 2 --epochs 30 --data coco128.yaml --weights yolov5s.pt**

Is this a bug (should I submit a bug report?), or am I doing something wrong?

**Comment by glenn-jocher** [comment]
@sandreel thanks for the bug report. This has been fixed a couple days ago in PR https://github.com/ultralytics/yolov5/pull/1566.

**Comment by sandreel** [comment]
True, after pulling again the results look correct. Thank you!
